setup.py

```diff
@@ -25,7 +25,7 @@
         "shapely",
         "structlog",
         "checksumdir",
-        "eodatasets",
+        "eodatasets>=0.19.2",
```

Comment:
`eodatasets` and `eodatasets3` are different libraries. I think we need both here until the old one is phased out.

Reply:
ok, I've changed setup.py to:
`        "eodatasets",
        "eodatasets3>=0.19.2",`